referral-form-data-process/package.json

```diff
@@ -15,7 +15,8 @@
     "jest": "^27.2.5",
     "prettier": "^2.4.1",
     "typescript": "^4.4.4",
-    "ts-jest": "^27.0.5"
+    "ts-jest": "^27.0.5",
+    "serverless": "^2.66.1"
```

Comment:
Would this lock this version of Serverless? 
Or do we need to remove the `^`?

Reply:
@Renny4Real this will be everything up to `<3.0.0`, so any compatible version of Serverless v2, which should be good for our use case 👍